pkg/pillar/cmd/domainmgr/containerd.go

```diff
@@ -36,44 +39,45 @@ type KeyValue struct {
 
 // getContainerPath return the path to the root of the container. This is *not*
 // necessarily the rootfs, which may be a layer below
-func getContainerPath(id string) string {
-	return path.Join(containersRoot, id)
+func getContainerPath(containerID string) string {
+	return path.Join(containersRoot, CtrdServicesNamespace, containerID)
 }
 
 // getContainerRootfs return the path to the root of the container filesystem
-func getContainerRootfs(id string) string {
-	return path.Join(getContainerPath(id), containerRootfsPath)
+func getContainerRootfs(containerID string) string {
+	return path.Join(getContainerPath(containerID), containerRootfsPath)
```

Comment:
As above, this might or might not be correct. It depends on what we are looking for. In general, though, the config will be above the rootfs.

Reply:
Corrected.

Reply:
Did you test it? The path to the container might not be the root of the path to the rootfs.